src/App.tsx

```diff
@@ -10,33 +13,25 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
```

Comment:
```suggestion
  const [selectedTabId, setSelectedTabId] = useState('tab-1');
```

maybe better use tabs[0].id